### ISSUE TITLE
PatchOrchestrationApplication healthy, but not installing updates.

### ISSUE THREAD

**Issue by laughtonsm** [Azure/service-fabric-issues#527]
Hi.

I have recently deployed the Patch Orchestration Application.  This is on a 3-node Azure Service Fabric cluster running version 6.0.211.9494.

The service appears to be installed correctly, and the only parameters that I adjusted from defaults was the WUFrequency, which I have set to 'Daily, 12:00:00'  However, when I hit the GetWindowsUpdateResults REST API, the results file is empty.  In addition, when I look in D:\PatchOrchestrationApplication directory I do not see any logs at all.  If I look at the Windows Update configuration in Server Manager, I can see that Windows Update is set to 'Download Updates Only, Using Windows Update', Last install is 'Never' and Last Check was at 3:38 this morning.  If I look at Updates, I can see that there are some available.

Just wondering if there is anything else I need to configure?  I have configured this across multiple clusters (including v5.7.217.9494) with similar bahaviour.

Just wondering if anyone has seen this, or has any guidance?
Cheers

Shaun

**Comment by novino** [comment]
Hi @laughtonsm, 

For clusters running SF version 5.6.220.9494 and above, POA logs would be stored along with SF runtime logs. You won't find any logs under D:\PatchOrchestrationApplication folder for your clusters.

'Download Updates Only, Using Windows Update' is correct, as POA automatically sets it to this level.

Could you list the KB numbers of the updates which you see as available but not installed. 

You can try increasing **WUOperationTimeOutInMinutes** to a higher value (300 mins instead of default 90), recently we've seen some cumulative monthly updates requiring a higher value than 90 mins depending on the type of VM you've.

WUOperationTimeOutInMinutes | Int (Default: 90) | Specifies the timeout for any Windows Update operation (search or download or install). If the operation is not completed within the specified timeout, it is aborted.
-- | -- | --

ref: https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-patch-orchestration-application#configure-the-app

You can share your clusterID , region for your azure cluster for us to have a look at your logs for exact issue which you might be facing.

**Comment by novino** [comment]
Hi Shaun, checking back to see if you were able to mitigate the issue.

**Comment by masnider** [comment]
Hi @laughtonsm - we haven't heard back from you on this. At this point we're going to close the issue. If you have more information about what you've tried that you can share with us, please just reopen the issue and let us know, or feel free to create a new issue if you prefer.

**Comment by laughtonsm** [comment]
Hi.  Apologies for the delay.  I've set the timeout to 300 now.  also, I adjusted the time that this happens to 19:00.  However, as before, patches are not installed each day.  One day randomly, the cluster started installing patches in the middle of the day - not according to the schedule.  Any advice would be welcome as today on the same cluster an update was attempted to be installed.  however the node wasn't disabled and the update failed.
